### PR TITLE
troubleshooting: add tip about --userns=auto and --userns=keep-id

### PR DESCRIPTION
**Pull request by eriksjolund**
#### Does this PR introduce a user-facing change?

```release-note
None
```

That `--userns=keep-id` blocks  `--userns=auto` can be a bit surprising . Let's document that. I first thought that this only happens when running rootless podman but I reproduced the same behaviour for rootful podman. The example in the troubleshooting tip uses rootless podman.

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -1435,3 +1435,48 @@ Using the default `overlay` storage driver, a `COPY`, `ADD`, or an I/O intensive
...
+On a Linux system
+
+1. Create a new user
+   ```
+   $ sudo useradd test
+   ```
+2. Start a login session
+   ```
+   $ sudo machinectl shell --uid=test
+   ```
```
Comment (Luap99) [inline troubleshooting.md:1454]: I think this should be dropped as it just adds noise without real benefit

Comment (eriksjolund) [reply troubleshooting.md]: Now fixed

**Review by Luap99 — APPROVED** [review]
LGTM

**Comment by eriksjolund** [comment]
I realised that the stopping of the containers does not matter. It's the deletion of the containers that matters.
Also `--nomap` culd also be mentioned. I updated the PR.

**Pull request merged**